utils/anautilities.py

```diff
@@ -875,6 +875,32 @@ def parseCalFile(filename=None):
 
     return (calDAC2Q_m, calDAC2Q_b)
 
+def parseArmDacCalFile(filename):
```

Comment:
Looks good except need function documentation here.  Please add then we can merge in.  See other functions in this file for examples.

Reply:
done